pandas/io/parsers.py

```diff
@@ -727,10 +731,7 @@ def __init__(self, f, engine=None, **kwds):
         self._make_engine(self.engine)
 
     def close(self):
-        try:
-            self._engine._reader.close()
-        except:
-            pass
+        self._engine.close()
```

Comment:
I shouldn't have. But this was only used by `CParserWrapper`, so I have put it there now in my last commit.